lms/static/scripts/frontend_apps/components/Breadcrumbs.js

```diff
@@ -32,7 +32,7 @@ export default function Breadcrumbs({
   const breadcrumbs = items.slice(0, -1);
   const currentItem = items[items.length - 1];
   return (
-    <ul className="Breadcrumbs hyp-u-layout-row">
+    <ul className="Breadcrumbs">
```

Comment:
It is my intent to extract the `Breadcrumbs` pattern/component to the shared package and so I have left it extant as-is; the only change here is to remove the dangling utility class here and include its effect in Breadcrumb's source CSS.